### PR TITLE
Add slack & mailing-list

### PR DESCRIPTION
**Pull request by sykp241095**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# TiDB Community
```
Comment (hi-rustin) [inline README.md:1]: Why we remove this title?

/hold

Comment (sykp241095) [reply README.md:1]: Duplicated with "TiDB Community Logo", the style is copied from: https://github.com/pingcap/tidb#readme

Comment (sykp241095) [reply README.md:1]: Preview: https://github.com/pingcap/community/tree/sykp241095-patch-4#readme

**Pull request merged**